modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -115,11 +115,12 @@ Node *EditorSceneImporterAssimp::import_scene(const String &p_path, uint32_t p_f
 	// Cannot remove pivot points because the static mesh will be in the wrong place
 	importer.SetPropertyBool(AI_CONFIG_IMPORT_FBX_PRESERVE_PIVOTS, true);
 	int32_t max_bone_weights = 4;
-	//if (p_flags & IMPORT_ANIMATION_EIGHT_WEIGHTS) {
```

Comment:
This is not needed, Let assimp import as many as it wants, and change SurfaceTool so you can choose there the amount of weights when creating it.